index.html

```diff
@@ -239,17 +240,15 @@ <h3>Latest Nightly Release</h3>
 						</tbody>

```

Comment:
Look two lines above this. That shows the "Latest Nightly Release". Unless I'm mistaken, you're not getting those with JS, are you. So they'll show strikethroughs if Jekyll fails, even if the JS succeeds, just like they do in https://media.discordapp.net/attachments/610531541889581068/1059327221019463750/image.png.

Reply:
You're right!